src/baseline_bots/parsing_utils.py

```diff
@@ -30,6 +30,21 @@ def dipnet_to_daide_parsing(
     :return: DAIDE style order string
     """
 
+    def replace_dipnet_loc(loc: str) -> str:
+        """
+        Replaces dipnet location with DAIDE location
+        E.g. BOT -> GOB
+             ENG -> ECH
+        """
+        if "BOT" in loc:
+            loc = loc.replace("BOT", "GOB")
+        if "ENG" in loc:
+            loc = loc.replace("ENG", "ECH")
+        if "GOL" in loc:
+            loc = loc.replace("GOL", "LYO")
```

Comment:
It's supposed to be the other way around: 

```suggestion
        if "LYO" in loc:
            loc = loc.replace("LYO", "GOL")
```

It probably didn't help that the issue author accidentally ordered them backwards.